src/main/java/strategy/BalanceStrategy.java

```diff
@@ -0,0 +1,11 @@
+package strategy;
+
+import db.ShopStorage;
+import model.FruitTransaction;
+
+public class BalanceStrategy implements OperationHandler {
```

Comment:
Why Strategy implements Handler? Please fix naming for all handlers
```suggestion
public class BalanceHandler implements OperationHandler {
```

Reply:
not fixed

Reply:
not fixed